build.properties

```diff
@@ -4,4 +4,4 @@
 #
 
 project.name=demo
-project.version=0.4.2
+project.version=0.4.4
```

Comment:
I would have expected a version bump to 0.4.3

Reply:
0.4.3 is already on the public repo ;
To avoid conflicts we need 0.4.4